commons-dlp/src/main/java/org/exoplatform/commons/dlp/domain/DlpPositiveItemEntity.java

```diff
@@ -42,9 +42,12 @@
     @Column(name = "KEYWORDS")
     private String keywords;
 
+    @Column(name = "ITEM_IS_EXTERNAL")
```

Comment:
We should not store it. 
User can become internal between the moment of the detection and the moment of the display.
External user information should be check at display only